huaweicloud/resource_huaweicloud_networking_secgroup.go

```diff
@@ -174,41 +233,77 @@ func resourceNetworkingSecGroupCreate(ctx context.Context, d *schema.ResourceDat
 
 func resourceNetworkingSecGroupRead(_ context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
 	config := meta.(*config.Config)
-	client, err := config.NetworkingV3Client(GetRegion(d, config))
+	region := GetRegion(d, config)
+	v1Client, err := config.SecurityGroupV1Client(region)
+	if err != nil {
+		return fmtp.DiagErrorf("Error creating HuaweiCloud networking v1 client: %s", err)
```

Comment:
it is security group cilent

Reply:
The difference between SecurityGroupV1Client() and NetworkingV1Client() is just whether there is a projectId in the URL.
They are all networking clients.